bindata/network/ovn-kubernetes/managed/common/004-config.yaml

```diff
@@ -127,6 +127,9 @@ data:
 {{- if (index . "V6JoinSubnet") }}
     v6-join-subnet="{{.V6JoinSubnet}}"
 {{- end }}
+
+    [logging]
+    libovsdblogfile=/var/log/ovnkube/libovsdb.log
```

Comment:
what of pod/container restart? Maybe logs need to be in a volume-mount? Maybe co-locate it with the dbs?